src/components/dashboard/forms/items/AdUnit/AdUnitTargeting.js

```diff
@@ -250,21 +251,21 @@ class AdUnitTargeting extends Component {
 							)
 						)}
 					</Grid>
-					<Grid item sm={12}>
```

Comment:
why is that change?
you had container with 2 items inside, now container with item, and item after the container